File: chat/src/main/java/com/jw/chat/GoChatError.java
package com.jw.chat;

public interface GoChatError {

	int ERROR_SERVER = 1;
	int ERROR_CLIENT_NET = 2;

	interface Login {
		int PASSWORD_ERROR = 100;
		int ACCOUNT_MISS = 101;
	}

	interface Register {
		int ACCOUNT_EXIST = 150;
	}
}

File: chat/src/main/java/com/jw/chat/GoChatError.kt
package com.jw.chat

interface GoChatError {

    interface Login {
        companion object {
            val PASSWORD_ERROR = 100
            val ACCOUNT_MISS = 101
        }
    }

    interface Register {
        companion object {
            val ACCOUNT_EXIST = 150
        }
    }

    companion object {
        val ERROR_SERVER = 1
        val ERROR_CLIENT_NET = 2
    }
}